src/main/java/com/iridium/iridiumskyblock/configs/Messages.java

```diff
@@ -147,6 +147,8 @@
     public String noSuchItem = "%prefix% &7You don't have this item!";
     public String successfullyBought = "%prefix% &7You have successfully bought %amount%x %item% &r&7for $%vault_cost% and %crystal_cost% Crystals.";
     public String successfullySold = "%prefix% &7You have successfully sold %amount%x %item% &r&7for $%vault_reward% and %crystal_reward% Crystals.";
+    public String successfullyBoughtBooster = "%prefix% &7You have successfully bought the %booster% Booster &r&7for $%vault_cost% and %crystal_cost% Crystals.";
```

Comment:
The &7&r isnt needed here? since its already &7?

Reply:
It is because the booster name can be bold for example. You could argue whether or not the &r should be in front of `Booster` though.

Reply:
But I have to say that it looked pretty good in chat